features/mesh/builtin-gateways/Item.feature

```diff
@@ -18,7 +18,7 @@ Feature: mesh / builtin-gateways / item
       KUMA_RULE_MATCHER_COUNT: 1
       KUMA_RULE_MATCH_COUNT: 2
       """
-    Given the URL "/meshes/default/meshgateways/gateway-1" responds with
+    Given the URL "/meshes/default/meshgateways/gateway-1.namespace" responds with
```

Comment:
Can we make `namespace` something else? I think it's a variable, right?
Here it looks like a constant in the template.

Reply:
oh yeah 2 tics, will switch up

Reply:
I changed it to `kuma-system`, thanks!